internal/cmd/feedback/command.go

```diff
@@ -0,0 +1,74 @@
+package feedback
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	cliv1 "github.com/confluentinc/ccloud-sdk-go-v2/cli/v1"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	pversion "github.com/confluentinc/cli/internal/pkg/version"
+)
+
+type command struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+func New(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "feedback",
+		Short:       fmt.Sprintf("Submit feedback for the %s.", pversion.FullCLIName),
+		Args:        cobra.NoArgs,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogin},
+	}
+
+	c := &command{
+		AuthenticatedCLICommand: pcmd.NewAuthenticatedCLICommand(cmd, prerunner),
+	}
+	cmd.RunE = c.feedback
+
+	return cmd
+}
+
+func (c *command) feedback(_ *cobra.Command, _ []string) error {
+	feedback, err := getFeedback(form.NewPrompt(os.Stdin))
+	if err != nil {
+		return err
+	}
+	if len(feedback) > 0 {
```

Comment:
This is a bit more precise. At first glance I thought `getFeedback()` was returning a slice...
```suggestion
	if feedback != "" {
```